lib/modelscan_api/bailo_modelscan_api/main.py

```diff
@@ -101,6 +130,7 @@ def scan_file(in_file: UploadFile, background_tasks: BackgroundTasks) -> dict[st
     finally:
         try:
             # Clean up the downloaded file as a background task to allow returning sooner.
+            # If using a temporary dir then this would happen anyway, but if Settings' download_dir evaluates then this is required.
```

Comment:
The /tmp only gets cleaned on restarts?

Reply:
Python's TemporaryDirectory creates a subdir of `/tmp` (e.g. `/tmp/zbc123xyz`) which is automatically cleaned up periodically https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory.cleanup